packages/neon-core/src/rpc/clients/NeoServerRpcClient.ts

```diff
@@ -258,6 +258,14 @@ export function NeoServerRpcMixin<TBase extends RpcDispatcherMixin>(
       const response = await this.execute(Query.validateAddress(addr));
       return response.isvalid;
     }
+
+    /**
+     * Get the amount of unclaimed GAS for a NEO address.
+     */
+    public async getUnclaimedGas(addr: string): Promise<number> {
+      const response = await this.execute(Query.getUnclaimedGas(addr));
+      return parseInt(response.unclaimed) / 100_000_000;
```

Comment:
This is possible unsafe if the initial parseInt is bigger than MAX_SAFE_INTEGER. Might also want to rethink about returning a number since that too might be unsafe but should be fine in this case.

```suggestion
      return BigInteger.fromNumber(response.unclaimed).div(100_000_000).toNumber();
```

(suggestion requires adding an import I think)

Reply:
I got a "property `toNumber()` does not exist on BigInteger". So instead I changed the return value to be BigInt. Which might be more suitable anyway, given that even after dividing by the decimals factor is might (although very unlikely) still exceed MAX_SAFE_INTEGER? Let me know what you think

Reply:
Wait I just realised we lose decimal precision in this case. We will need a better construct around this but for now Fixed8 will work. Look at Fixed8.fromRawNumber for your usage.

Reply:
Done